<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookSelect</class>
 <widget class="QDialog" name="BookSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Your Preffered Books</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <layout class="QHBoxLayout" name="CentralWidget">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="LeftPanel">
       <item>
        <widget class="QLabel" name="LblSidebar">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/images/images.qrc">:/images/Sidebar.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="RightPanel">
       <item>
        <widget class="QLabel" name="LblProgress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TxtSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Search Songbooks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="LstBooks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ProgressLayout"/>
       </item>
       <item>
        <widget class="QLabel" name="LblDetails">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BottomLayout">
         <item>
          <widget class="QLabel" name="LblNothing">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnProceed">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>PROCEED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnCancel">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
